run.py

```diff
@@ -172,15 +184,90 @@ def main(gtk_context):
         (subjects_dir / "fsaverage5").symlink_to(orig_subject_dir / "fsaverage5")
         (subjects_dir / "fsaverage6").symlink_to(orig_subject_dir / "fsaverage6")
 
+    bids_filter_file_path = gtk_context.get_input_path("bids-filter-file")
+    if bids_filter_file_path:
+        paths = list(Path("input/bids-filter-file").glob("*"))
+        log.info("Using provided PyBIDS filter file %s", str(paths[0]))
+        config["bids-filter-file"] = str(paths[0])
+
+    previous_work_zip_file_path = gtk_context.get_input_path("work-dir")
+    if previous_work_zip_file_path:
+        paths = list(Path("input/work-dir").glob("*"))
+        log.info("Using provided fMRIPrep intermediate work file %s", str(paths[0]))
+        unzip_dir = FWV0 / "unzip-dir2"
```

Comment:
For maintainability, is there a more descriptive name than dir2?

Reply:
Good suggestions.  I changed them.